webpack/components/Content/ContentPage.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Grid, Col, Row, Form, FormGroup } from 'react-bootstrap';
+import Search from '../../components/Search/index';
+import ContentTable from './ContentTable';
+
+const ContentPage = props => (
```

Comment:
The way you have things here works fine, but you can destructure the props object and assign to variables instead of calling props each time

for example: `const ContentPage = ({ header, onSearch, etc... }) => (`